sdk/python/tests/unit/permissions/auth/server/conftest.py

```diff
@@ -47,16 +106,36 @@ def temp_dir():
 
 
 @pytest.fixture
-def feature_store(temp_dir, auth_config):
+def feature_store(temp_dir, auth_config, applied_permissions):
     print(f"Creating store at {temp_dir}")
-    return _default_store(str(temp_dir), auth_config)
+    return _default_store(str(temp_dir), auth_config, applied_permissions)
 
 
 @pytest.fixture
 def server_port():
     return free_port()
 
 
+@pytest.fixture(
+    scope="module",
+    params=[
```

Comment:
since it's `Optional`, can you add the `None` value?

Reply:
Changed it, no none option